### PR TITLE
Misc cleanups related to "make check"

### PR DESCRIPTION
**Pull request by clebergnu**
Just a collection of misc cleanups and fixes related to "make check".

### CODE REVIEW

File: examples/jobs/passjob_html.py [hunk 1]
```diff
@@ -6,8 +6,7 @@
 from avocado.core.suite import TestSuite
 
 config = {'run.references': ['examples/tests/passtest.py:PassTest.test'],
-          'job.run.result.html.enabled': 'on',
-          'job.run.result.html.open_browser': True}
```
Comment (beraldoleal) [inline examples/jobs/passjob_html.py:10]: Thanks. I guess it is a nice feature, but for the entire job.

File: examples/jobs/passjob_html_browser.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+
+import sys
+
+from avocado.core.job import Job
+from avocado.core.suite import TestSuite
+
+config = {'run.references': ['examples/tests/passtest.py:PassTest.test'],
+          'job.run.result.html.enabled': 'on',
+          'job.run.result.html.open_browser': True}
```
Comment (beraldoleal) [inline examples/jobs/passjob_html_browser.py:10]: ohhh and here are again. :/

**Review by beraldoleal — APPROVED** [review]
LGTM

**Review by willianrampazzo — APPROVED** [review]

File: examples/jobs/passjob_html_browser.py (hunk 2 above)
Comment (clebergnu) [reply examples/jobs/passjob_html_browser.py:10]: > ohhh and here are again. :/

Right, but this doesn't get run on the selftests! :)

**Pull request merged**